sw/device/lib/testing/test_framework/ottf_common.ld

```diff
@@ -262,7 +262,7 @@ SECTIONS {
   .non_volatile_scratch _non_volatile_scratch_start (NOLOAD) : ALIGN(2048) {
     KEEP(*(.non_volatile_scratch))
     KEEP(*(.non_volatile_scratch.*))
-    . = _non_volatile_scratch_size;
+    . = _non_volatile_scratch_start + _non_volatile_scratch_size;
```

Comment:
```suggestion
    . = ABSOLUTE(_non_volatile_scratch_start + _non_volatile_scratch_size);
```

This style should make it compatible with both linkers.

Reply:
But actually, there are various *_end values already calculated, so you can use those directly:
```suggestion
    . = ABSOLUTE(_non_volatile_scratch_end);
```

This is what I did in https://github.com/lowRISC/opentitan/pull/20320